graylog2-server/src/main/java/org/graylog2/commands/journal/JournalShow.java

```diff
@@ -75,7 +75,7 @@ private void appendSegmentDetails(KafkaJournal journal, StringBuffer sb) {
             sb.append("\t\t").append("Segment ").append(i++).append("\n");
             sb.append("\t\t\t").append("Base offset: ").append(segment.baseOffset()).append("\n");
             sb.append("\t\t\t").append("Size in bytes: ").append(segment.size()).append("\n");
-            sb.append("\t\t\t").append("Created at: ").append(new DateTime(segment.created(), DateTimeZone.UTC)).append("\n");
+            sb.append("\t\t\t").append("Created at: ").append(new DateTime(segment.size(), DateTimeZone.UTC)).append("\n");
```

Comment:
I somehow doubt that this is correct. 😉

Reply:
Segment.created() has been moved from a public method to a private method.

Reply:
@pbr0ck3r This doesn't make it more correct.

What's the meaning of a timestamp generated out of the size of a segment on disk? 😉

Reply:
@joschi I wasn't inferring that it was correct. I was just stating that from looking at the Segment library code that it is no longer a public method and has been moved to private. It does not look like they have created any helper/alternative methods to get the create datetime.

Reply:
.size() is indeed not correct. Corrected it.